metadata_tools/util.py

```
@@ -410,70 +512,104 @@ def sclk_format_count(fields, format):
+
+    Returns:
+        bool: True if the observation is exluded.
+    """
+    if not exceptions:
+        return False
+
+    id = get_observation_id(observation)
+    for exception in exceptions:
+        # check for config function
+        if exception.isidentifier():
+            fn = getattr(config, exception)
```

Flake8 (7.3.0)

[error] undefined name 'config'

(F821)